ebpf-io/ebpf.io-website
# feat: update ebpf libraries component

This pull request brings design update to eBPF libraries component. 

Related to issue #498 

**Steps to check:**
- open [preview](https://deploy-preview-528--ebpfio.netlify.app/infrastructure/#ebpf-libraries)
- make sure that everything looks and feels as expected
